frontend/javascripts/oxalis/model/helpers/compaction/compact_save_queue.ts

```diff
@@ -5,7 +5,10 @@ function removeAllButLastUpdateTracingAction(updateActionsBatches: Array<SaveQue
   // This part of the code removes all entries from the save queue that consist only of
   // one updateTracing update action, except for the last one
   const updateTracingOnlyBatches = updateActionsBatches.filter(
-    (batch) => batch.actions.length === 1 && batch.actions[0].name === "updateTracing",
+    (batch) =>
+      batch.actions.length === 1 &&
+      (batch.actions[0].name === "updateSkeletonTracing" ||
+        batch.actions[0].name === "updateVolumeTracing"),
```

Comment:
@fm3 do you know whether this compaction code is still correct now? the old code removed dropped updateTracing actions because only the newest one would be relevant in a batch. do `updateSkeletonTracing` and `updateVolumeTracing` encode the same information?

Reply:
No, unfortunately, one of each needs to be sent

Reply:
ok, I fixed it in 171d853d23.